guides/advanced_guides/tokenization.md

```diff
@@ -0,0 +1,31 @@
+# Tokenization
+
+**Tokenization** is the act of taking a sentence or phrase and splitting it into smaller units of language, called tokens. It is the first step of document indexing in the Meilisearch engine, and is a critical factor in the quality of search results.
+
+Breaking sentences into smaller chunks requires understanding where one word ends and another begins, making tokenization a highly complex and language-dependant task. MeiliSearch's solution to this problem is a **modular tokenizer** that follows different processes, called **pipelines**, based on the type of character it detects. This allows it to function on datasets from multiple languages with minimal setup.
+
+Currently, the MeiliSearch tokenizer has full support for four languages: English, Chinese, Japanese, and Korean. It can also effectively tokenize any language that uses the Latin alphabet.
+
+## The Meili Tokenizer
```

Comment:
```suggestion
## Deep Dive in the MeiliSearch Tokenizer
```
Simple suggestion as this goes deeper than what a common user would want to know.